tests/system/small/bigquery/test_json.py

```diff
@@ -139,3 +139,14 @@ def test_json_extract_from_string():
 def test_json_extract_w_invalid_series_type():
```

Comment:
please add a similar test to improve the code coverage

Reply:
done